### DISCUSSION TITLE
Superuser Tab No Response

### DISCUSSION THREAD

**Discussion by masteruni** [bmax121/APatch#366] [Q&A] [answered]
It seems I successfully installed and patched my boot and flashed it through TWRP recovery. But when I go to Superuser Tab, it keeps on loading and not showing anything. I'm using Evolution-X ROM with Android 14.

**Comment by marat2509** [comment]
Most likely the problem occurs in the process of getting the list of installed applications, which happens cyclically and consequently you don't see the list of applications in the tab.

Try searching for the appropriate permission in the application's permissions

**Comment by pomelohan** [comment, marked as answer]
It's a known issue, check releaseNotes:
`We have noticed that some older devices (mainly those with Linux kernel versions less than 5) may experience unlimited loading of SuperUser Screen, and even some applications may not be able to use ROOT permissions. This is a known issue and is currently under investigation.`
